### DISCUSSION TITLE
next/image always prepends imgix path even if absolute url is provided

### DISCUSSION THREAD

**Discussion by agcty** [vercel/next.js#18390] [Ideas]
# Bug report

## Describe the bug
When you add a custom loader and path to your next.config.js, the path is prepended to every image.

```
images: {
    deviceSizes: [320, 420, 768, 1024, 1200],
    loader: "imgix",
    path: "https://my-imgix-dev.imgix.net/",
},
```

```javascript
const imgSrc = "https://some-external-image.com/random.png";

<Image
     src={imgSrc}
     width={300}
     height={300}
     alt={alt}
     className={classNames("rounded-lg grid-item", {
     "opacity-75": loadingState.isLoading,
     })}
   />
```

the src of the image is: https://my-imgix-dev.imgix.net/https://some-external-image.com/random.png

## Expected behavior

Just the external url should be set without prepending the path:

https://some-external-image.com/random.png

**Comment by 0leksiy** [comment]
If you set a loader and a path you can only use the Image component to get images from that cloud provider.

**Comment by matamatanot** [comment]
How is this

```
images: {
    loader: "imgix",
    path: "",
}
```

**Reply by uinstinct** [reply]
No, it still does not work.
Can you please send me your configuration for **cloudinary** or **imgix**?

**Reply by brodyd795** [reply]
This worked perfectly for me! Thank you!

**Reply by kb1995** [reply]
This worked for me as well! Thanks

**Reply by dbismut** [reply]
This doesn't work anymore on Next v12.

**Reply by ujnimz** [reply]
Get this warning Next v13
```
warn  - Invalid next.config.js options detected: 
  - The value at .images.path must be 1 character or more but it was 0 characters.
```

**Comment by ujnimz** [comment]
I did this to solve my issue. 

```
var relativeUrl = new URL(absoluteUrl);

<Image
      layout='responsive'
      src={relativeUrl.pathname}
      alt='alternativeText'
      height={400}
      width={300}
/>
```
